cafs/ipfs/config.go

```diff
@@ -118,16 +117,16 @@ func (cfg *StoreCfg) InitRepo(ctx context.Context) error {
 // MoveIPFSRepoOntoPath moves the ipfs repo from wherever it is,
 // indicated by the store config, to live on the given path
 // this changes the path in the given config struct
-func MoveIPFSRepoOntoPath(o *StoreCfg, path string) error {
-	if path == "" {
+func MoveIPFSRepoOntoPath(repoPath, newPath string) error {
```

Comment:
This is equivalent to just `os.Rename`. If either argument to that function is an empty string you'll get the error "no such file or directory". How about calling that directly from `RunMigrations`.

Reply:
Yea that sounds good. We've minimized/changed the responsibilities of `MoveIPFSRepoOntoPath` so much that it has slimmed down so much as to be unnecessary.